### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by koomekevo**
This pull request has been created by the setup-project branch to achieve the following:

- Add dependencies such as prop-types, redux, redux-logger, and redux-thunk.
- Add devDependencies such as Eslint and Stylelint.
- Add functionality and styling.
- Add tests.  
- Deploy live demo to Netlify.

### CODE REVIEW

File: src/__tests__/__snapshots__/snapshots.test.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import Crypto from '../../components/Crypto';
+import EFTs from '../../components/EFTs';
+import Home from '../../components/Home';
+import Forex from '../../components/Forex';
+import Loading from '../../components/Loading';
+import Majors from '../../components/Majors';
+import Stocks from '../../components/Stocks';
+
+describe('test screenshots for all components', () => {
+//   const getState = () => ({ price: 0.145947, ticket: 'EUR/USD' });
```
Comment (Mihndim2020) [inline src/__tests__/__snapshots__/snapshots.test.js]: - [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

File: src/components/Stocks.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  return (
+    <section className="stocks-page">
+      <div className="top-bar">
+        <button
+          type="button"
+          className="go-back"
+          onClick={() => navigate('/')}
+        >
+          <i className="chevron left icon" />
+        </button>
```
Comment (Mihndim2020) [inline src/components/Stocks.js:42]: - [ ] As it is requirement for this project, which also improves the user experience of your app, kindly ensure  there's a link(`"Back" button`) in the `Detail page` that you can click on to go to the `Home page`. This button is absent in your app, please kindly fix that. See screenshot of the requirements below;

backbutton

File: src/components/Majors.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  return (
+    <section className="majors-page">
+      <div className="top-bar">
+        <button
+          type="button"
+          className="go-back"
+          onClick={() => navigate('/')}
+        >
+          <i className="chevron left icon" />
+        </button>
```
Comment (Mihndim2020) [inline src/components/Majors.js:43]: - [ ] As it is requirement for this project, which also improves the user experience of your app, kindly ensure  there's a link(`"Back" button`) in the `Detail page` that you can click on to go to the `Home page`. This button is absent in your app, please kindly fix that. See screenshot of the requirements below;

backbutton

File: src/components/Forex.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  return (
+    <section className="forex-page">
+      <div className="top-bar">
+        <button
+          type="button"
+          className="go-back"
+          onClick={() => navigate('/')}
+        >
+          <i className="chevron left icon" />
+        </button>
```
Comment (Mihndim2020) [inline src/components/Forex.js:33]: - [ ] As it is requirement for this project, which also improves the user experience of your app, kindly ensure  there's a link(`"Back" button`) in the `Detail page` that you can click on to go to the `Home page`. This button is absent in your app, please kindly fix that. See screenshot of the requirements below;

backbutton

File: src/components/EFTs.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  return (
+    <section className="efts-page">
+      <div className="top-bar">
+        <button
+          type="button"
+          className="go-back"
+          onClick={() => navigate('/')}
+        >
+          <i className="chevron left icon" />
+        </button>
```
Comment (Mihndim2020) [inline src/components/EFTs.js:43]: - [ ] As it is requirement for this project, which also improves the user experience of your app, kindly ensure  there's a link(`"Back" button`) in the `Detail page` that you can click on to go to the `Home page`. This button is absent in your app, please kindly fix that. See screenshot of the requirements below;

backbutton

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @koomekevo, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: Your app is running free of `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below and under the review._

- [ ] Great video presentation, but as it is a requirement for this project, please  it will be nice to highlight;
- all your project features,
- a piece of the code that you find interesting or are proud of making.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :partying_face: 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**